initial start on assistant docs

Dannyholt opened the pull request.

nancyhild commented on docs/assistant/cloud/libraries/overview.md:
```diff
@@ -0,0 +1,17 @@
+Assistant also supports the templates and saved content of most popular page editors including Divi, Visual Composer and Elementor.
+
+
+* [Adding assets overview](adding-assets/overview.md)
```
Consistency: KB uses the imperative, not gerund, in titles with procedures to distinguish them from conceptual articles.

La-Fi replied:
My grammar is atrocious so I'm at a loss here.  @nancyhild, will you please make a suggestion for updated copy?

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-cloud.md:
```diff
@@ -0,0 +1,20 @@
+sidebar_label: Assistant Pro Cloud
+---
+
+### How to ​​Create a New Library From Assistant Pro Cloud
```
Title is "Adding Assets" but first procedure is how to create a new library--at least needs a conceptual intro about where you add assets.

nancyhild replied:
How to Create -> Create

FROM Assistant Pro  Cloud or IN Assistant Pro Cloud?

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-cloud.md:
```diff
@@ -0,0 +1,20 @@
+
+### How to ​​Create a New Library From Assistant Pro Cloud
+
+Enter the library’s name then click the enter icon.  Premium users can also make the library private by clicking the lock icon (or you can make libraries private under library settings). 
```
Premium users....    put this sentence into a note.

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-cloud.md:
```diff
@@ -0,0 +1,20 @@
+<source src="/video/assistant/libraries--create-library.webm" type="video/webm" />
+</video>
+
+## How to Add Items to a Library From Assistant Pro Cloud
```
How to Add -> Add

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+title: Adding Assets via the Assistant plugin
+sidebar_label: Assistant Plugin
+---
+
```
Needs an intro that explains why the first procedure is "Create a Library"

La-Fi replied:
Intro:

In Assistant Pro, all of our assets are organized into libraries.

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+To create a new library from the Assistant Plugin:
+
+* Open the Libraries App
```
Numbered procedure with period at the end of each step.

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+You can add page content like posts, pages, Beaver Builder Templates, etc. from the [Content App](../../../plugin/apps/content.md), [Media App](../../../plugin/apps/media.md), or the [Libraries App](../../../plugin/apps/libraries.md).
+
+## Add Content via the Content App
```
H3: ###

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+<source src="/video/assistant/libraries--add-assets-content-app.webm" type="video/webm" />
+</video>
+
+## Add Content via the Media App
```
H3: ###

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+## Add Content via the Media App
+
+You can add .jpg, .png, and .svg graphics to Assistant Pro from the Media App.  By default, you can’t add .svg graphics to WordPress media storage, so being able to add .svg graphics to WordPress via the Assistant Plugin can save you the trouble of using ftp to add .svg graphics to your WordPress sites.
```
This is repeating info from elsewhere, and it appears to have left out .gif. Better to use a link.

nancyhild replied:
By default....

NH: This seems like a sales feature so I'd recommend making a tip out of it.

Dannyholt replied:
Fixed `.gif`

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+## Add Content via the Content App
+
+To add content from the Content App, click the piece of content you want to add, then select the Add to Library button.  You’ll be prompted to select which library you’d like to add the content to.
```
Numbered procedure.
(Makes it easier to scan the page to find instructions.)

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+## Adding Content via the Libraries App
+
+You can add content, media, colors and customizer (theme) settings from the [Libraries App](../../../plugin/apps/libraries.md).
+
```
Procedure?

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+### Add Posts, Pages or Templates to libraries from the Libraries App
+
+* Open the Libraries App
```
Numbered steps with periods.

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+You can add content, media, colors and customizer (theme) settings from the [Libraries App](../../../plugin/apps/libraries.md).
+
+### Add Posts, Pages or Templates to libraries from the Libraries App
```
H4 so ####

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+<img src="https://plchldr.co/i/800x300?&bg=f6f6f6&fc=656565&text=Placeholder" />
+
+### Add a Color to an Assistant Pro Library From the Libraries App
```
H4 so ####

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+### Add a Color to an Assistant Pro Library From the Libraries App
+
+* Open the Libraries App
```
Numbered steps with periods.

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+### Add Theme Settings to an Assistant Pro Library From the Libraries App
+
+This will save your site’s customizer settings.  
```
This will save -> This procedure saves

nancyhild commented on docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md:
```diff
@@ -0,0 +1,73 @@
+
+This will save your site’s customizer settings.  
+
+* Open the Libraries App
```
Numbered steps.

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/add-single-items.md:
```diff
@@ -0,0 +1,28 @@
+sidebar_label: Add single items
+---
+
+## Post types, blocks, Beaver Builder & 3rd party page builder templates
```
## Add a post type, block, Beaver Builder or third-party page builder template

NH: "post type" or "post"?

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/add-single-items.md:
```diff
@@ -0,0 +1,28 @@
+
+## Post types, blocks, Beaver Builder & 3rd party page builder templates
+
+* Navigate to the piece of content you’d like to import to your site from the Libraries App
```
* Navigate to the piece of content  ->  1. Navigate to the content

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/add-single-items.md:
```diff
@@ -0,0 +1,28 @@
+* Navigate to the piece of content you’d like to import to your site from the Libraries App
+* Select Create and Edit or Import from the Create New drop-down menu
+	* Create and Edit will import the content and open the content for you to edit
+	* Import will add the content to the backend of your site and the [Content App](../../../plugin/apps/content.md)
```
* Import will add --> * Import adds

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/add-single-items.md:
```diff
@@ -0,0 +1,28 @@
+
+* Navigate to the piece of content you’d like to import to your site from the Libraries App
+* Select Create and Edit or Import from the Create New drop-down menu
+	* Create and Edit will import the content and open the content for you to edit
```
will import the content and open the content for you to edit-> imports the content and opens it for editing

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/add-single-items.md:
```diff
@@ -0,0 +1,28 @@
+
+## Images
+
+* Navigate to the image you want to import from your Libraries App
```
Numbered steps

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/entire-libraries.md:
```diff
@@ -0,0 +1,13 @@
+---
+id: entire-libraries
+title: Adding entire libraries to a site
```
Adding -> Add

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/entire-libraries.md:
```diff
@@ -0,0 +1,13 @@
+sidebar_label: Entire Libraries
+---
+
+In addition to adding single items to Assistant Pro libraries, you can also import entire libraries.  To import an entire library:
```
In addition to adding single items to Assistant Pro libraries, you can also import entire libraries.  ->  You can import an entire Assistant Pro library.

nancyhild commented on docs/assistant/cloud/libraries/using-library-assets/entire-libraries.md:
```diff
@@ -0,0 +1,13 @@
+
+In addition to adding single items to Assistant Pro libraries, you can also import entire libraries.  To import an entire library:
+
+* Navigate to the library you’d like to import to your site from the [Libraries App](../../../plugin/apps/libraries.md)
```
Numbered procedure.

nancyhild commented on docs/assistant/cloud/libraries/organize-assets.md:
```diff
@@ -0,0 +1,59 @@
+sidebar_label: Organizing Assets
+---
+
+Assistant Pro Collections work in a similar way to WordPress post categories.  You can add more than one collection to any creative asset (posts, pages, images …) and that creative asset will show up under each collection.  This way of organizing files is preferred over folders, because files can be cross referenced in multiple places.
```
This is the first time I'm seeing the term "Assistant Pro Collections". Define it? Or introduce the concept higher up in the documentation?

La-Fi replied:
{ The term should be Library Collection defined on the [Terminology](https://deploy-preview-201--angry-gates-bead98.netlify.app/assistant/#terminology) page. }

Updated Copy: 

Assistant Pro’s Library Collections work in a similar way to WordPress post categories.  You can add more than one collection to any creative asset (posts, pages, images …) and that creative asset will show up under each collection.  This way of organizing files is preferred over folders, because files can be cross referenced in multiple places.

nancyhild commented on docs/assistant/cloud/libraries/organize-assets.md:
```diff
@@ -0,0 +1,59 @@
+
+To create a library collection from the Assistant Pro Cloud:
+
+* Select the library you’d like to add a collection to
```
Numbered steps.

Dannyholt replied:
Fixed in PR

nancyhild commented on docs/assistant/cloud/libraries/view-edit-assets.md:
```diff
@@ -0,0 +1,25 @@
+---
+id: view-edit-assets
+title: View & Edit Assets
```
View and Edit

(for search and consistency with headings)

nancyhild commented on docs/assistant/cloud/libraries/view-edit-assets.md:
```diff
@@ -0,0 +1,25 @@
+
+## View and edit assets in the Assistant Plugin
+
+To view the attributes of and edit creative assets in the Assistant Plugin, click on the asset from your library.  Clicking the asset will open a window that will reveal the asset’s attributes and allow you to perform various actions including:
```
will open a window that will reveal the asset's attributes and allow -> opens a window that displays the asset's attributes and allows

Dannyholt replied:
Fixed this in my PR

nancyhild commented on docs/assistant/cloud/libraries/delete-library.md:
```diff
@@ -0,0 +1,19 @@
+
+You can delete an Assistant Pro library from either the Assistant Pro Cloud or the Assistant Plugin.
+Delete an Assistant Pro Library from The Assistant Pro Cloud
+To delete a library from the Assistant Pro Cloud, select the library you’d like to delete, then from the bottom of the settings window, select Delete This Library.
```
To delete a library:
1. In Assistant Pro Cloud, select the library you want to delete.
2. Click **Delete this library** below the settings.

Dannyholt replied:
Fixed in PR

nancyhild reviewed (changes requested):
In general:
* Clearly distinguish between procedural content and conceptual content. Users search for procedural content first, resort to conceptual content when that doesn't work. Use imperatives for procedural content and avoid passives everywhere (with exceptions in cases of grammatical focus)
* Turn procedural content into numbered steps when possible. This makes it much easier for users to scan the page to find it and makes the sequence of steps more obvious. 
* Distinguish UI controls and labels by putting them in bold.
* Single source of information: present details about what constitutes assets in one place and refer back to that.
* Avoid "will", hammered into me by an excellent editor of my own work. This helps to simplify sentence processing load (along with avoiding passives).

Dannyholt commented on docs/assistant/getting-started/what-can-i-do.md:
```diff
@@ -0,0 +1,35 @@
+sidebar_label: What can I do with Assistant Pro?
+---
+
+Assistant Pro is a cloud sharing application that will help you keep your WordPress assets organized and easily accessible.  It links to your WordPress sites using a plugin.  The plugin offers a suite of apps that let you perform common tasks using a frontend sidebar.
```
Fixed in PR

La-Fi commented on docs/assistant/getting-started/what-can-i-do.md:
```diff
@@ -0,0 +1,35 @@
+## Assistant WordPress Plugin
+
+Allows you to perform common WordPress tasks more efficiently using a sidebar on the frontend of your website.
+
```
{ I added a note to let people know you can use the plugin without the cloud, and the new text in brackets indicates that you can’t use the cloud without the plugin. } 

Updated Copy:

NOTE: You don’t need to use the Assistant Pro Cloud to enjoy the WordPress plugin. The Assistant Pro Cloud extends the uses of the WordPress plugin.

A cloud storage platform that connects to the WordPress sites you’ve installed the Assistant WordPress plugin on. It allows you to easily access, share and organize design assets across multiple sites [via the Assistant Plugin].

La-Fi commented on docs/assistant/getting-started/what-can-i-do.md:
```diff
@@ -0,0 +1,35 @@
+---
+id: what-can-i-do
+title: What can I do with Assistant Pro?
```
{ This section should be omitted since listing the common tasks would be redundant of the what is section. }

Dannyholt replied:
Ok I've removed this article.

La-Fi commented on docs/assistant/getting-started/overview.md:
```diff
@@ -0,0 +1,12 @@
+sidebar_label: Overview
+slug: /assistant/getting-started/
+---
+
```
{my response is a little complicated to put here in the notes so I'm going to give a[ link to my Google Doc with notes}(https://docs.google.com/document/d/1bMYk5KhDZj7lYMS0cY0TQ0VMU2g5NFl4DIeMahnOcx8/edit#bookmark=kix.doup1u7pz4qn)}.

La-Fi commented on docs/assistant/getting-started/signup.md:
```diff
@@ -0,0 +1,10 @@
+sidebar_label: Signup
+---
+
+You can sign up for the Assistant Pro Cloud at [Assistant.Pro](https://assistant.pro/).
```
Updated Copy:

You sign up for the Assistant Pro Cloud at [Assistant.Pro](https://assistant.pro/).

La-Fi commented on docs/assistant/getting-started/account-types.md:
```diff
@@ -0,0 +1,13 @@
+
+There are three Assistant Pro account types: Free, Personal and Team. 
+ ​​
+* The Assistant WordPress Plugin is free to all users  
```
{to me it seems better to point people to the comparison table than explain the account types.}

Updated Copy:

There are three Assistant Pro account types: Free, Personal and Team.​​  The Assistant WordPress Plugin is free to all users.  The Assistant Pro Cloud App has both free and premium options.  There are two premium options, one for individuals and one for teams.
You can change your account type from the Account Tab under Billing Information. You can compare prices and features on this comparison table.

La-Fi commented on docs/assistant/getting-started/account-types.md:
```diff
@@ -0,0 +1,13 @@
+There are three Assistant Pro account types: Free, Personal and Team. 
+ ​​
+* The Assistant WordPress Plugin is free to all users  
+* The Assistant Pro Cloud App has both free and premium options
```
{there is only one plugin. The proper term is Assistant Pro Plugin.}

Updated Copy:

You can download the Assistant Pro Plugin from your Assistant.Pro Cloud dashboard. Click the three dots to toggle the dropdown menu, and select Download Plugin.

La-Fi commented on docs/assistant/getting-started/installation.md:
```diff
@@ -0,0 +1,23 @@
+
+## Download from Assistant.pro
+
+You can download the Assistant Pro WordPress Plugin from your Assistant.Pro Cloud dashboard. Click the three dots to toggle the dropdown menu, and select Download Plugin.
```
{there is only one plugin. The proper term is Assistant Pro Plugin.  Maybe we need to clarify the plugin always stays the same, users Cloud accounts have options.}

Updated Copy:

You can download the Assistant Pro Plugin from your Assistant.Pro Cloud dashboard. Click the three dots to toggle the dropdown menu, and select Download Plugin.

La-Fi commented on docs/assistant/getting-started/account-types.md:
```diff
@@ -0,0 +1,13 @@
+ ​​
+* The Assistant WordPress Plugin is free to all users  
+* The Assistant Pro Cloud App has both free and premium options
+* There are two premium options, one for individuals and one for teams
```
{ The main difference in premium account types is that one is for individuals and one is for teams.  Maybe this could be made more clear with a second level of bullets. }

Updated Copy:
* The Assistant WordPress Plugin is free to all users  
* The Assistant Pro Cloud App has both free and premium options
* There are two premium options
            -Premium Accounts for Individual Users
            -Premium Accounts for Team Users

La-Fi commented on docs/assistant/getting-started/link-to-assistant-cloud.md:
```diff
@@ -0,0 +1,9 @@
+sidebar_label: Link to Assistant Pro Cloud
+---
+
+From the Assistant plugin’s sidebar, navigate to the Libraries App. There you can connect your site to the Assistant Pro Cloud. 
```
{ There is only one plugin and it's always free.  You can link free accounts to the cloud .  }

La-Fi commented on docs/assistant/plugin/apps/media.md:
```diff
@@ -0,0 +1,15 @@
+
+From the media app, you can browse and sort media or upload new media.  
+
+You can also add media to the Assistant Pro cloud from the Media App.  See Adding Content From the Media App for details.
```
Assistant Pro Cloud

La-Fi commented on docs/assistant/plugin/apps/libraries.md:
```diff
@@ -0,0 +1,27 @@
+sidebar_label: Libraries
+---
+
+The Libraries App is where you connect your WordPress sites to Assistant Pro so you can add and access assets to and from the Assistant Pro Cloud. 
```
You connect multiple sites.

La-Fi commented on docs/assistant/plugin/apps/libraries.md:
```diff
@@ -0,0 +1,27 @@
+
+## Launch Assistant Pro Cloud
+
+There is a link to the Assistant Pro Cloud in the Libraries App of the WordPress Plugin. If you are connected to Assistant Pro, from the Libraries App you will see your user avatar. Click your avatar and select Launch Cloud from the dropdown menu.
```
This is a common misconception in Nancy's notes... there is only one plugin avalible.  There are multiple levels of Cloud user accounts.  Maybe this is something that needs to be clarified so people understand?

La-Fi commented on docs/assistant/plugin/overview.md:
```diff
@@ -0,0 +1,15 @@
+sidebar_label: Overview
+slug: /assistant/plugin/
+---
+
```
There is only one plugin.

La-Fi commented on docs/assistant/cloud/overview.md:
```diff
@@ -0,0 +1,32 @@
+* Personal
+* Private
+* Shared
+* Team
```
Updated Copy:

Private or Public (Private libraries are only available on Premium accounts)
Personal, Shared, or Team (Team Libraries are only available for Team account users)

La-Fi commented on docs/assistant/cloud/account-settings/profile-settings.md:
```diff
@@ -0,0 +1,19 @@
+sidebar_label: Public Profile Settings
+---
+
+Your public profile is part of the Assistant Pro Community.  Public profile settings can be set from the Assistant Pro Cloud dash under Account>General.
```
New Copy: 

If you don't want to make my profile available to the community, it can be disabled.  By default is is enabled.

La-Fi commented on docs/assistant/cloud/account-settings/transfer-ownership.md:
```diff
@@ -0,0 +1,13 @@
+
+## Transfer Library Ownership from the Assistant Pro Cloud
+
+You can transfer a library from the Assistant Pro Cloud dash from the settings page near the bottom of the settings page options.
```
To transfer a library from the Assistant Pro Cloud dash, go to the settings page and click the Transfer Library option.

La-Fi commented on docs/assistant/cloud/account-settings/create-manage-team-libraries.md:
```diff
@@ -0,0 +1,21 @@
+
+You need to have a Team Account in order to create teams.  You can upgrade to a Team Account from the Billing Tab on your Account Page.
+
+Teams are created from your Account Page.  The teams you manage are accessed by toggling a 
```
To Create a Team
- Click on the Account tab
- Select "Create a Team"
- Name your team
- Click the "Create Team" button

To Add Team Members and Assign Permissions
- Select "Members" > "Add Members" > "Invite Members"
- Enter the email addresses and permission levels for your team members

La-Fi commented on docs/assistant/cloud/account-settings/create-manage-team-libraries.md:
```diff
@@ -0,0 +1,21 @@
+<source src="/video/assistant/accoun-settings--create-team.webm" type="video/webm" />
+</video>
+
+Teams display on the Libraries Page sidebar under Shared Libraries.
```
The library called Team Library under Testing Team with the big red arrow is a team library....  "Testing Team" in blue is a team.

Dannyholt commented:
## Introduction & What can I do
* Moved the “What is” content from the Introduction page to the What can I.. page with bullets - it makes more sense now.
* Added Quick Start to the Introduction page.


## Link Assistant Cloud
Replaced **Link** on the article **Link to Assistant Pro Cloud** with **Connect to...** as the button on the Assistant Pro website uses Connect.

**You can see a preview here - https://deploy-preview-201--angry-gates-bead98.netlify.app/assistant**

---

@La-Fi 

I think we need some more context for the app articles i.e. Home, Media, Updates etc. Some basics on all the features per app.

**For Example:**

**Home App**
* Tells you the page you’re currently viewing
* How many posts, pages, comments and updates are on your site.
* View recently added media (images, videos) and lets you quickly access the media app to edit the image or video.
* Recent Posts - lets you view recent posts created with a dropdown menu to filter the content i.e. posts, pages, reusable blocks, templates and themes layouts.

If possible, can you do something like this for each app and for the App Settings doc, please. :D

Also, there are only a few other reviews of Nancy's left which haven't been responded to. Can you give this look, please.

---

@nancyhild 

Can you give this another pass, please to make sure everything is ok.

La-Fi commented on docs/assistant/cloud/libraries/overview.md:
```diff
@@ -0,0 +1,17 @@
+slug: /cloud/libraries/
+---
+
+You can add colors and images (.jpg, .png, .svg .giff) as well as templates shared by the community to your libraries from the Assistant Pro Cloud.
```
...See [Assistant Pro Cloud Overview](https://deploy-preview-201--angry-gates-bead98.netlify.app/assistant/) for details about what creative assets are and y for how to view these assets in the Libraries app of your plugin.
cloud/

La-Fi commented:
@Dannyholt Here's the new text for more context for the app articles:
https://docs.google.com/document/d/1bMYk5KhDZj7lYMS0cY0TQ0VMU2g5NFl4DIeMahnOcx8/edit#bookmark=id.ai11fl30m0bs

Dannyholt commented on docs/assistant/overview.md:
```diff
@@ -0,0 +1,93 @@
+* **Team** - Libraries and their permissions can be shared across a team of users.
+
+:::info
+The basic Team plan includes 3 personal user accounts. Additional users can be added to the team.
```
Yup there is a charge ~ Additional users can be added for $5 per month each.

nancyhild replied:
ok then the wording stands.

Dannyholt commented on docs/assistant/getting-started/installation.md:
```diff
@@ -0,0 +1,36 @@
+
+## Download from WordPress.org
+
+You can download the Assistant Plugin from the [Wordpress.org](https://wordpress.org/plugins/assistant/) plugin repository.
```
@nancyhild I updated this section last night with the following:

```markdown
## Install from WordPress.org

1. Download the Assistant Plugin from the [Wordpress.org](https://wordpress.org/plugins/assistant/) plugin repository.
2. Once downloaded, visit your website's WordPress Admin Panel, go to **Plugins > Add New** and click the **Upload plugin** button at the top of the screen.
3. Click **Browse** and select the downloaded zip file on your local system or just drag the file on top of the **Browse** button.
4. Click **Install** now.
```

nancyhild replied:
@Dannyholt Looks good.

Dannyholt commented on docs/assistant/plugin/apps/overview.md:
```diff
@@ -0,0 +1,24 @@
+slug: /assistant/plugin/apps/
+---
+
+The Assistant WordPress plugin adds a sidebar to the frontend pages of your site.  The sidebar lets you easily perform tasks that you would normally need to access from the back end of your site. The sidebar is toggled on by clicking the floating pencil icon in the upper right corner of your site’s frontend pages.
```
@nancyhild should this paragraph end with a colon or period?

Dannyholt commented on docs/assistant/plugin/apps/updates.md:
```diff
@@ -0,0 +1,15 @@
+sidebar_label: Updates
+---
+
+The Updates App allows you to review all available updates for your (plugins & themes).
```
I replaced "software" with "plugin and theme".

Dannyholt commented on docs/assistant/getting-started/what-can-i-do.md:
```diff
@@ -0,0 +1,38 @@
+* Import, export, and save WordPress assets to a file sharing cloud app.
+* Manage WordPress content (pages, posts, custom post types, and media), WooCommerce content, Beaver Builder templates, and Beaver Themer layouts.
+* Support for third-party page builder templates, including Divi, Visual Composer, and Elementor.
+
```
I just removed the images.

Dannyholt commented on docs/assistant/cloud/libraries/overview.md:
```diff
@@ -0,0 +1,17 @@
+sidebar_label: Overview
+slug: /cloud/libraries/
+---
+
```
I'm leaving Libraries where they're because its an Assistant Pro feature and the Assistant plugin simply integrates into it.

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+To add collections to a creative asset:
+
+1. Select the asset.
```
```suggestion
1. Click on an asset to open its settings.
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+To add collections to a creative asset:
+
+1. Select the asset.
+2. Enter collection names into the sidebar dialogue field. 
```
```suggestion
2. In the **Collections** field, start typing a collection name. Collection names that you've already used in that library will appear as suggestions.
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+![Create collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--3.jpg)
+
+## Assign Collections to a creative asset from Assistant Pro Cloud
```
```suggestion
## Assign Collections to a creative asset from Assistant Pro Cloud or the Assistant plugin
```
As far as I can see, the procedure is identical so you can combine this section with the next one.

Dannyholt replied:
I'd prefer to keep them separated.

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+![Assign assets to collections from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--4.jpg)
+
+## Assign Collections to a creative asset from the Assistant Plugin
```
If you agree this procedure is identical to the previous one, you can delete lines 48-53.

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+## Sort by Collection from the Assistant Pro Cloud
+
+​​By default libraries are sorted by file type. Once you have added collections, you can also sort by collection.
```
```suggestion
​​By default, library assets are sorted by file type. Once you have added collections, you can also sort by collection.
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+​​By default libraries are sorted by file type. Once you have added collections, you can also sort by collection.
+
+View a list of a library’s collections by clicking the collections tab.
```
```suggestion
To view a library’s assets grouped by collection, open a library and click the **Type** dropdown at the top of the page, then choose **Collection**.  
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+## Sort by Collection from the Assistant Plugin
+
+To sort a library by collection from the Assistant Plugin:
```
```suggestion
To sort a library's assets by collection from the Assistant Plugin:
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+To sort a library by collection from the Assistant Plugin:
+
+* Select **View by Collection** from the dropdown menu.
```
```suggestion
* Click the **View by** dropdown and choose **Collection**.
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+* Select **View by Collection** from the dropdown menu.
+
+![Sort by collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--7.jpg)
```
```suggestion
![Sort assets by collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--7.jpg)
```

nancyhild commented on docs/assistant/cloud/collections.md:
```diff
@@ -0,0 +1,75 @@
+
+View a list of a library’s collections by clicking the collections tab.
+
+![Sort by collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--6.jpg)
```
```suggestion
![Sort assets by collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--6.jpg)
```

nancyhild commented on docs/assistant/plugin/apps/libraries.md:
```diff
@@ -0,0 +1,101 @@
+
+## Launch Assistant Pro Cloud
+
+The Assistant Pro Cloud can be launched from within the **Libraries App** providing quick access. To launch the Assistant Pro Cloud from the **Libraries App**:
```
```suggestion
You can quickly launch Assistant Pro Cloud in a browser from within Assistant plugin's **Libraries App**. 

To launch Assistant Pro Cloud from the **Libraries App**:
```

nancyhild commented on docs/assistant/plugin/apps/libraries.md:
```diff
@@ -0,0 +1,101 @@
+The Assistant Pro Cloud can be launched from within the **Libraries App** providing quick access. To launch the Assistant Pro Cloud from the **Libraries App**:
+
+1. Open the **Libraries App**.
+2. Click your avatar to deploy the dropdown menu.
```
```suggestion
2. Click your avatar in the upper right corner.
```
```suggestion
2. Click your avatar to deploy the dropdown menu.
```

Dannyholt commented on docs/assistant/cloud/libraries/using-assets/add-single-items.md:
```diff
@@ -0,0 +1,31 @@
+
+## Theme Settings
+
+Navigate to the theme setting you want to import and click **Apply to Theme**. You have the option to preview the theme settings before importing.
```
Added this to my patch, but probably need to add something about their old settings.
